vsg/__main__.py

```diff
@@ -146,10 +146,7 @@ def read_configuration_files(dStyle, commandLineArguments):
     dConfiguration = dStyle
     if commandLineArguments.configuration:
         for sConfigFilename in commandLineArguments.configuration:
-            try:
```

Comment:
I can not remember why I had this split.  `open_configuration_file` does not raise any exceptions.  The only difference is `cofmmandLineArguments.junit`...so I would assume if that did not exist then only call `open_configuration_file` without it.

Maybe it was test induced?

Reply:
I also had to update a test class with `junit = None`, so I guess that's why you had it split.